tests/migration/reboot_to_upgrade.pm

```diff
@@ -32,8 +33,7 @@ sub run {
         set_var('BOOT_HDD_IMAGE', 0) unless check_var('ARCH', 'aarch64');
     }
     assert_script_run "sync", 300;
-    type_string "reboot -f\n";
+    power_action('reboot', textmode => 1, keepconsole => 1);
```

Comment:
According to #4597, could you keep `-f` for reboot if don't mind.
Other LGTM.

Reply:
Right, I removed that commit altogether as `power_action()` can't cope with options like `-f`.